src/data/map.js

```diff
@@ -1058,7 +1058,7 @@ const getNests = async (minLat, maxLat, minLon, maxLon, nestFilterExclude = null
     return null;
 };
 
-const getSearchData = async (lat, lon, id, value) => {
+const getSearchData = async (lat, lon, id, value, iconStyle) => {
```

Comment:
Stop letting server resolve iconStyle. 🤷

Reply:
There is no fallback logic on the server side. This will lead to form not resolving correctly.

Reply:
How else will the server know what style the client chose?

Reply:
No need. Send pokemon data to client and let client do the work. See my other PR on quest rewards.

Reply:
For example: https://github.com/versx/MapJS/pull/117/files#diff-095e8a9344f3ac4f7d1799f877e63e05L225